lib/evss/reference_data/configuration.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+module EVSS
+  module ReferenceData
+    class Configuration < EVSS::Configuration
+      PROXY_OPTS = {
+        proxy: {
```

Comment:
Does this belong here or somewhere more generally usable?

Reply:
I only envision this proxy setting being used for the `reference_data_service`.  That said, I think it appropriate to define inside the `reference_data_service` `configuration`.  Do you think other services will want this as well?